tests/integration_tests/test_mlflow.py

```diff
@@ -268,6 +268,26 @@ def test_nest_trials(tmpdir: py.path.local) -> None:
     assert all(set(r.data.metrics.keys()) == {"value"} for r in child_runs)
 
 
+@pytest.mark.parametrize("n_jobs", [2, 4, 6])
```

Comment:
Do we need all these values?
In my environment, we the current change increase the test time from 1.9 sec to 2.5 sec. If we remove `6` from `test_multiple_jobs` and `test_track_in_mlflow_decorator`, it the test time will be about 2.0 sec.

And I also confirmed that `n_jobs=2` and `n_jobs=4` failed with the current master.

```suggestion
@pytest.mark.parametrize("n_jobs", [2, 4])
```

Reply:
I thought setting n_jobs to 6 is not such an expensive task. So, I added `6` just in case. To be honest, this test sometimes passes the test when n_jobs is 2 on my machine. But it would be enough to test only 2 and 4.